python/pip_install/private/generate_whl_library_build_bazel.bzl

```diff
@@ -232,19 +232,28 @@ def generate_whl_library_build_bazel(
         if p.startswith("@"):
             continue
 
-        os, _, cpu = p.partition("_")
+        abi, _, tail = p.partition("_")
+        os, _, cpu = tail.partition("_")
+        if not abi.startswith("cp"):
+            # The first item is not an abi
+            abi, os, cpu = "", abi, os
+
+        constraint_values = [
+            "@platforms//cpu:{}".format(cpu),
+            "@platforms//os:{}".format(os),
+        ]
+        if abi:
+            minor_version = int(abi[len("cp3"):])
+            constraint_values.append("@rules_python//python/config_settings:is_python_3.{}".format(minor_version))
```

Comment:
I think the hard-coded `@rules_python` part of this string is incorrect if repo remapping is in use?